presto-product-tests/README.md

```diff
@@ -28,3 +28,73 @@ Product tests are not run by default. To start them use run following command:
 ```
 java -jar target/presto-product-tests-*-executable.jar --config-local file://`pwd`/tempto-configuration-local.yaml
 ```
```

Comment:
Can you also add that Blackhole connector has to be configured? One the test depends on it.